BreastCancer/Startup/APHAppDelegate.m

```diff
@@ -55,20 +55,20 @@
 /*********************************************************************************/
 #pragma mark - Initializations Options
 /*********************************************************************************/
-static NSString* const  kStudyIdentifier            = @"studyname";
-static NSString* const  kAppPrefix                  = @"studyname";
-static NSString* const  kVideoShownKey              = @"VideoShown";
-static NSString* const  kConsentPropertiesFileName  = @"APHConsentSection";
+static NSString* const  kStudyIdentifier					= @"studyname";
+static NSString* const  kAppPrefix							= @"studyname";
+static NSString* const  kVideoShownKey						= @"VideoShown";
+static NSString* const  kConsentPropertiesFileName			= @"APHConsentSection";
 
```

Comment:
I looks these string are misaligned after the patch. Do you have a non-default tab space setting?